draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -168,7 +168,9 @@ The goals do not include:
   : The rate-limit fields may be returned in both
     Successful and non Successful responses.
```

Comment:
Is there a reason why "Successful" is capitalized?  Also, I would have expected the opposite to be written "non-successful".

Reply:
Clarified adding a reference to SEMANTICS.

```suggestion
    successful (see Section 15.3 of {{SEMANTICS}}) and non-successful responses.
```

Reply:
@darrelmiller clarified, thanks!